.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,61 +1,51 @@
 ---
 name: 🐛 Bug report
 about: Create a report to help us improve
-
 ---
+## Bug report
+<!--- Verify first that your issue is not already reported on GitHub -->
+<!--- Also test if the latest release and devel branch are affected too -->
+<!--- This form is processed by Ansibot (machine), complete *all* sections as described -->
 
-<!---
-Verify first that your issue/request is not already reported on GitHub.
-THIS FORM WILL BE READ BY A MACHINE, COMPLETE ALL SECTIONS AS DESCRIBED.
-Also test if the latest release, and devel branch are affected too.
-ALWAYS add information AFTER (OUTSIDE) these html comments.
-Otherwise it may end up being automatically closed by our bot. -->
-
-##### SUMMARY
-<!--- Explain the problem briefly -->
+### Summary
+<!--- Explain the problem briefly below -->
 
-##### ISSUE TYPE
- - Bug Report
```

Comment:
Does Ansibullbot rely on the ISSUE TYPE section for anything?

Reply:
I think so, yes. So we may have to ensure that we cover this.